pkg/agent/openflow/client.go

```diff
@@ -317,6 +322,15 @@ func (c *client) Initialize(roundInfo types.RoundInfo, config *config.NodeConfig
 	return connCh, c.initialize()
 }
 
+func (c *client) InstallExternalFlows(nodeIP net.IP, localSubnet net.IPNet) error {
+	flows := c.snatFlows(config.UplinkOFPort, config.BridgeOFPort, nodeIP, cookie.SNAT)
+	flows = append(flows, c.l3ToExternalFlows(nodeIP, localSubnet, config.BridgeOFPort, cookie.SNAT)...)
+	if err := c.flowOperations.AddAll(flows); err != nil {
+		return fmt.Errorf("failed to install flows to enable local Pod to communicate to the external addresses: %v", err)
```

Comment:
Pod -> Pods

To be shorter, you might just say sth. like "flows for external communication".